### ISSUE TITLE
HA 0.115 appears to break this integration

### ISSUE THREAD

**Issue by robmarkcole** [robmarkcole/HASS-Google-Vision#9]
HA 0.115 appears to break this integration:
```
Platform error image_processing.google_vision - Error relocating /usr/local/lib/python3.8/site-packages/grpc/_cython/cygrpc.cpython-38-arm-linux-gnueabihf.so: backtrace: symbol not found
```

**Comment by robmarkcole** [comment]
Assign this issue to @giloris1

**Comment by robmarkcole** [comment]
Reported resolved in 0.115.4 possibly due to https://github.com/home-assistant/core/pull/40678
